noggin/tests/unit/controller/test_user_otp.py

```diff
@@ -256,7 +257,7 @@ def test_user_settings_otp_disable_ipaerror(
         )
         result = client.post(
             "/user/dummy/settings/otp/disable/",
-            data={"token": dummy_user_with_2_otp[1].uniqueid},
+            data={"token": dummy_user_with_2_otp[1]},
```

Comment:
Uh, why this change?

Reply:
I'm not sure about this one myself. I'll give it a go with that change backed out.